opencensus/exporters/trace/stackdriver/stackdriver_exporter.h

```diff
@@ -15,62 +15,22 @@
 #ifndef OPENCENSUS_EXPORTERS_TRACE_STACKDRIVER_STACKDRIVER_EXPORTER_H_
 #define OPENCENSUS_EXPORTERS_TRACE_STACKDRIVER_STACKDRIVER_EXPORTER_H_
 
-#include <memory>
-#include <string>
-#include <vector>
-
-#include "google/devtools/cloudtrace/v2/tracing.grpc.pb.h"
-#include "include/grpc++/grpc++.h"
-#include "opencensus/trace/exporter/span_data.h"
-#include "opencensus/trace/exporter/span_exporter.h"
+#include "absl/strings/string_view.h"
 
 namespace opencensus {
 namespace exporters {
 namespace trace {
 
-class StackdriverExporter
-    : public ::opencensus::trace::exporter::SpanExporter::Handler {
+class StackdriverExporter {
```

Comment:
Are these changes to make it match the stats exporter?

Reply:
Yes, particularly making the public API smaller (just `Register`)